### ISSUE TITLE
mentor: claiming a solution for review

### ISSUE THREAD

**Issue by NobbZ** [exercism/exercism.io#3850]
As a mentor one might stumble over solutions where on sees on the first glance that it will take some time to review.

It would be nice if I could hit a button that tells other mentors, that I am currently doing my work and am spending the necessary time.

Such claims should expire after a while, perhaps when claiming even ask 30 mins to 2h.

**Comment by iHiD** [comment]
That's a great idea. Will do it once all the bugs (etc) are fixed.

**Comment by NobbZ** [comment]
As we can't see the age of a comment, I do not consider this a viable option.

**Comment by NobbZ** [comment]
Oh, it could work. I checked, and I can see the age of comments, but not of the solution, I mixed things up here.

So yes, this could be a possibility, but on the other hand side might cause expectations on the students side that we might not able to hold.

Lets say I put a message that I will come back with more details later in an hour, than one of my children falls and we need to drive to the hospital. Or my wife wants me to do the dishes. Just regular emergencies... The student would wait for my reply, but it doesn't happen.
